Formula/terraforming.rb

```diff
@@ -0,0 +1,113 @@
+  end
+
+  resource "aws-sdk-elasticache" do
+    url "https://rubygems.org/gems/aws-sdk-elasticache-1.1.0.gem"
+    sha256 "1cf9a2b9124b3490ee6aa2d5328b0144f70de45565def4605cbea5b26753d5af"
+  end
+
+  resource "aws-sdk-elasticloadbalancing" do
+    url "https://rubygems.org/gems/aws-sdk-elasticloadbalancing-1.1.0.gem"
+    sha256 "a819b8427733f06cd83fe41a912e953b5503a61d857d3b9f9548d7801c3e9534"
+  end
+
+  resource "aws-sdk-elasticloadbalancingv2" do
+    url "https://rubygems.org/gems/aws-sdk-elasticloadbalancingv2-1.3.0.gem"
+    sha256 "e578163969e152dca6ee538dc958218d66e8cad00d8dd8332609bb95ea7566cc"
+  end
+
+  resource "aws-sdk-iam" do
+    url "https://rubygems.org/gems/aws-sdk-iam-1.3.0.gem"
+    sha256 "e724ac129db6d5c7e9ec43c1fb1ca47abb1490a9e8013d0be9501f1660324ba6"
+  end
+
+  resource "aws-sdk-kms" do
+    url "https://rubygems.org/gems/aws-sdk-kms-1.2.0.gem"
+    sha256 "4a76cbd7a9e605e52d1851fd38c4121a29e5aa7c18f0a6eab816842598765fa9"
+  end
+
+  resource "aws-sdk-rds" do
+    url "https://rubygems.org/gems/aws-sdk-rds-1.4.0.gem"
+    sha256 "a7116634bb8645f8c288748f44207f97364e8358b4f8743a9df16cf1b3a8d637"
+  end
+
+  resource "aws-sdk-redshift" do
+    url "https://rubygems.org/gems/aws-sdk-redshift-1.1.0.gem"
+    sha256 "f4622f0c4a6f3b04f5b3e6b27bca4baccca94a2db6708f60da6df9defe7e8d86"
+  end
+
+  resource "aws-sdk-route53" do
+    url "https://rubygems.org/gems/aws-sdk-route53-1.3.0.gem"
+    sha256 "98299ba6d3d8c9a54fdab9dec71a2ac409277bdcd187fce4d7012caa4d5aff46"
+  end
+
+  resource "aws-sdk-s3" do
+    url "https://rubygems.org/gems/aws-sdk-s3-1.5.0.gem"
+    sha256 "6efcfc8249a1de4b4626c603ae5190b30b3762fbc5ed6536caa483a4c262328e"
+  end
+
+  resource "aws-sdk-sns" do
+    url "https://rubygems.org/gems/aws-sdk-sns-1.1.0.gem"
+    sha256 "253cb65bebcfd6d2d6650899335c4b71a2d977adad53eeae21467800892cd79b"
+  end
+
+  resource "aws-sdk-sqs" do
+    url "https://rubygems.org/gems/aws-sdk-sqs-1.2.0.gem"
+    sha256 "ab48397ecb46909607afb3e7f1fa171fcc7e9d9337fac53a2825b3241ca51a9d"
+  end
+
+  resource "multi_json" do
+    url "https://rubygems.org/gems/multi_json-1.12.2.gem"
+    sha256 "5dcc0b569969f3d1658c68b5d597fcdc1fc3a34d4ae92b4615c740d95aaa51e5"
+  end
+
+  resource "thor" do
+    url "https://rubygems.org/gems/thor-0.20.0.gem"
+    sha256 "b47dad86e151c08921cf935c1ad2be4d9982e435784d6bc223530b62a4bfb85a"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    resources.each do |r|
+      r.verify_download_integrity(r.fetch)
+      system "gem", "install", r.cached_download, "--no-document",
+             "--install-dir", libexec
+    end
+    system "gem", "build", "terraforming.gemspec"
+    system "gem", "install", "--ignore-dependencies",
+           "terraforming-#{version}.gem"
+    bin.install libexec/"bin/terraforming"
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+  end
+
+  test do
+    output = shell_output("#{bin}/terraforming help ec2")
```

Comment:
This should do something other than run help. See `faas-cli` for an example of how to mock a connection.

Reply:
The example that you point to is spawning a TCP server listening on `localhost`, in this formula's case the software will try to reach Amazon's server and there seems to be no way to change the target url.